common/src/main/java/com/sequenceiq/cloudbreak/logger/MDCBuilder.java

```diff
@@ -133,7 +143,7 @@ public static String getFieldValue(Object o, String field) {
             privateStringField.setAccessible(true);
             return privateStringField.get(o).toString();
         } catch (Exception ignored) {
-            return "undefined";
+            return null;
```

Comment:
if we change this to null instead of undefined then we should switch everything to null